File: SMI-SDK/.NET SDK/pt/sapo/gis/trace/configuration/AppenderConfigProperties.cs
/*
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */
namespace pt.sapo.gis.trace.configuration
{
    using System;
    using System.Collections.Generic;
    using System.Linq;

    /**
     *
     * @author GIS Team
     */
    public class AppenderConfigProperties
    {

        protected IDictionary<String, String> properties = new Dictionary<String, String>();

        public AppenderConfigProperties(List<pt.sapo.gis.trace.configuration.Appender.Property> properties)
        {
            if (properties != null)
            {
                this.properties = properties.ToDictionary(p => p.Name, p => p.Content);
            }
        }
    }
}

File: SMI-SDK/Java SDK/src/pt/sapo/gis/trace/configuration/AppenderConfigProperties.java
/*
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */
package pt.sapo.gis.trace.configuration;

import java.util.HashMap;
import java.util.List;
import java.util.Map;
import pt.sapo.gis.trace.configuration.Appender.Property;

/**
 *
 * @author GIS Team
 */
public class AppenderConfigProperties {
    
    protected Map<String, String> propertiesMap = new HashMap<String, String>();
    
    public AppenderConfigProperties(List<Property> properties){         
        if(properties != null){            
            for(Property p : properties)
                propertiesMap.put(p.getName(), p.getContent());
        }
    }
}
